src/test/java/com/training/api/tests/shorex/ShorexTests.java

```diff
@@ -0,0 +1,30 @@
+package com.training.api.tests.shorex;
+
+import com.training.basetest.ApiBaseTest;
+import com.training.utilities.RestEngine;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ShorexTests extends ApiBaseTest {
+
+    @BeforeEach
+    public void startup(){
+        headerMap.put(map.get("gaAppKeyHeaderName"),map.get("gaAppKeyHeaderValue"));
+        headerMap.put(map.get("gaContentTypeHeaderName"),map.get("gaContentTypeHeaderValue"));
```

Comment:
Lets generalise this name to `rcclContentTypeHeaderName ` and `rcclContentTypeHeaderValue `